x-pack/plugins/ml/public/application/services/job_service.js

```diff
@@ -329,26 +329,17 @@ class JobService {
     // create a deep copy of a job object
     // also remove items from the job which are set by the server and not needed
     // in the future this formatting could be optional
+
     const tempJob = cloneDeep(job);
 
     // remove all of the items which should not be copied
     // such as counts, state and times
-    delete tempJob.state;
-    delete tempJob.job_version;
+    delete tempJob.calendars;
     delete tempJob.data_counts;
-    delete tempJob.create_time;
-    delete tempJob.finished_time;
-    delete tempJob.last_data_time;
     delete tempJob.model_size_stats;
-    delete tempJob.node;
-    delete tempJob.average_bucket_processing_time_ms;
-    delete tempJob.model_snapshot_id;
-    delete tempJob.open_time;
-    delete tempJob.established_model_memory;
-    delete tempJob.calendars;
-    delete tempJob.timing_stats;
     delete tempJob.forecasts_stats;
-    delete tempJob.assignment_explanation;
+    delete tempJob.state;
```

Comment:
As above, did you mean to include these changes? Looks like they are part of the `exclude_generated` work.

Reply:
This is now reverted https://github.com/elastic/kibana/pull/88880/commits/311892828ba8ad9bf92efe643c97c43d7c0439a1